x/concentrated-liquidity/msg_server_test.go

```diff
@@ -294,11 +294,41 @@ func (s *KeeperTestSuite) TestCollectSpreadRewards_Events() {
 				PositionIds: tc.positionIds,
 			}
 
+			// Add spread rewards to the pool's accum so we aren't just claiming 0 rewards.
+			// Claiming 0 rewards is still a valid message, but is not as valuable for testing.
+			s.AddToSpreadRewardAccumulator(validPoolId, sdk.NewDecCoin(ETH, sdk.NewInt(1)))
+
+			// Determine expected rewards from all provided positions without modifying state.
+			expectedTotalSpreadRewards := []sdk.Coin(nil)
+			cacheCtx, _ := s.Ctx.CacheContext()
+			for _, positionId := range tc.positionIds {
+				spreadRewardsClaimed, _ := s.App.ConcentratedLiquidityKeeper.PrepareClaimableSpreadRewards(cacheCtx, positionId)
+				for _, spreadReward := range spreadRewardsClaimed {
+					for i, expectedSpreadReward := range expectedTotalSpreadRewards {
+						if expectedSpreadReward.Denom == spreadReward.Denom {
+							expectedTotalSpreadRewards[i].Amount = expectedSpreadReward.Amount.Add(spreadReward.Amount)
+							goto nextSpreadReward
+						}
+					}
+					expectedTotalSpreadRewards = append(expectedTotalSpreadRewards, spreadReward)
+				}
+			nextSpreadReward:
+			}
```

Comment:
```suggestion
			expectedTotalSpreadRewards := sdk.Coins(nil)
			cacheCtx, _ := s.Ctx.CacheContext()
			for _, positionId := range tc.positionIds {
				spreadRewardsClaimed, _ := s.App.ConcentratedLiquidityKeeper.PrepareClaimableSpreadRewards(cacheCtx, positionId)
				expectedTotalSpreadRewards = expectedTotalSpreadRewards.Add(spreadRewardsClaimed...)
			}
```

Reply:
Done here https://github.com/osmosis-labs/osmosis/pull/5548/commits/2d2eb5d703df42a7347e5d6a3e329bdb762c84ab

But had to change proto so we use Coins instead of []Coin